commands/bootstrap/index.js

```diff
@@ -38,6 +38,13 @@ class BootstrapCommand extends Command {
   initialize() {
     const { registry, npmClient = "npm", npmClientArgs = [], mutex, hoist, nohoist } = this.options;
 
+    if (npmClient === "pnpm") {
+      throw new ValidationError(
+        "EWORKSPACES",
+        "Bootstrapping with pnpm is not supported. Use pnpm directly to manage dependencies (https://pnpm.io/cli/install)."
```

Comment:
Please align to match formatting of error for `add`, e.g. when it comes to the URL